ce/marketing/solution-uninstall-order.md

```diff
@@ -28,6 +28,42 @@ To delete Dynamics 365 from an instance, first open the Marketing setup wizard a
 
 Marketing solution uninstall order:
 
+Real-time part of marketing:
```

Comment:
Let's use the public facing name here. "Customer journey orchestration" or "Moments based marketing" instead of "Real-time part of marketing".